mlflow/langchain/__init__.py

```diff
@@ -542,8 +542,8 @@ def load_retriever(persist_directory):
             .. Note:: Experimental: This parameter may change or be removed in a future
                                     release without warning.
         streamable: A boolean value indicating if the model supports streaming prediction. If
-            True, the model must implement `stream` method. If None, the model's streamability
-            is inferred from the model type. Default to `None`.
+            True, the model must implement `stream` method. If None, we set streamable=True if
+            the model has `stream` attribute. Default to `None`.
```

Comment:
I avoid using `we` or `you` for user-facing messages.

Reply:
Updated!